packages/components/scss/bootstrap_imports.scss

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Bootstrap v4.6.2 (https://getbootstrap.com/)
+ * Copyright 2011-2022 The Bootstrap Authors
+ * Copyright 2011-2022 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
+ */
+
+/* Deephaven manually imports only the necessary bootstrap files */
+
+@import 'bootstrap/scss/functions';
+
+// patch the functions
+@import './bootstrap_override_functions.scss';
+
+@import 'bootstrap/scss/variables';
+@import 'bootstrap/scss/mixins';
+
+// patch the mixins
+@import './bootstrap_override_mixins_background_variant.scss';
+@import './bootstrap_override_mixins_buttons.scss';
+@import './bootstrap_override_mixins_forms.scss';
+@import './bootstrap_override_mixins_text_emphasis.scss';
+
+@import 'bootstrap/scss/root';
+@import 'bootstrap/scss/reboot';
+@import 'bootstrap/scss/type';
+// @import 'bootstrap/scss/images';
```

Comment:
Maybe group the ones we're intentionally not importing together and just say "we're not importing these as they're not needed" or something?

Reply:
Bootstrap import order is important which is why they were left in their original positions. That said, I can also add that detail to the comment telling devs to consult the original order if they need to add any back